.github/workflows/rust.yml

```diff
@@ -0,0 +1,43 @@
+name: Rust
+
+on:
+  push:
+    branches: [ master ]
+    paths:
+      - 'sdk/rust/**'
+      - 'lib/linux/**'
+      - 'lib/mac/**'
+      - 'lib/raspberry-pi/**'
```

Comment:
add jetson and beaglebone

Reply:
:facepalm: Got too excited by how easy this was

Reply:
Should we also add the resources directory to these? It would make sense to re-run tests if the contexts change right?